packages/react-vapor/src/components/select/examples/ServerSideSelectExamples.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import * as _ from 'underscore';
+
+import {IReactVaporExampleState} from '../../../../docs/Reducers';
+import {withServerSideProcessing} from '../../../hoc/withServerSideProcessing/withServerSideProcessing';
+import {FilterBoxSelectors} from '../../filterBox/FilterBoxSelectors';
+import {IItemBoxProps} from '../../itemBox/ItemBox';
+import {selectWithFilter} from '../hoc/SelectWithFilter';
+import {selectWithInfiniteScroll, SelectWithInfiniteScrollProps} from '../hoc/SelectWithInfiniteScroll';
+import {ISingleSelectOwnProps, SingleSelectConnected} from '../SingleSelectConnected';
+
+const TOTAL_ENTRIES = 5000;

```

Comment:
Do we really need to set this value?

Reply:
yes, its used to determine if the `hasMore` prop of the InfiniteScroll

Reply:
I discovered that the json-server exposes the total count in the request headers at `x-total-count` so I used this instead of the hardcoded value.